### PR TITLE
Add some queries and mutations and dummy data

### PR DESCRIPTION
**Pull request by bkzhang**
Errors and Error in `schema.js` is formatted the way it is so that the `winners` mutation can look like 
```
mutation {
  winners(users:["@brian.zhang"]) {
    errors {
      message
    }
  }
}
```

else it would look like
```
mutation {
  winners(users:["@brian.zhang"]) {
    message
  }
}
```
which doesn't show what message is supposed to indicate.

Comment if theres a cleaner way pls 🙏

### CODE REVIEW

File: graphql/error.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { builtinModules } from "module";
+
+class Error {
+  constructor(message) {
+    this.message = message
+  }
+}
+
+modules.export = Error
```
Comment (CapySloth) [inline graphql/error.js]: No new line?!

Comment (bkzhang) [reply graphql/error.js]: anarchy!

File: graphql/error.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { builtinModules } from "module";
```
Comment (Ipkhchan) [inline graphql/error.js]: what is this?

File: graphql/error.js (hunk 1 above)
Comment (Ipkhchan) [reply graphql/error.js]: also, should be module.exports not modules.export.

File: graphql/root.js [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+
+function topOfLeaderboard(numUsers) {
+  return ["@ivan.chan", "@brian.zhang", "@nashua.luk"];
+}
+
+function bottomOfLeaderboard(numUsers) {
+  return ["@dummy", "@data"];
+}
+
+function topWinStreaks(numUsers) {
```
Comment (nashanator17) [inline graphql/root.js]: We should make a default value instead of taking in arguments. For ex: always return top 10 winstreaks

Comment (bkzhang) [reply graphql/root.js]: depends on how we plan to implement it, out of this PRs scope

File: graphql/error.js (hunk 2 above)
Comment (bkzhang) [reply graphql/error.js]: bad copy pasta

**Comment by bkzhang** [comment]
> I don't think we need an Error class and custom return object actually. GraphQL seems to pass back error message if it encounters any.
> Screen Shot 2019-04-04 at 9 27 01 AM

I changed it to userErrors as I don't think graphQL will return user related errors like wrong slack handle

**Pull request merged**